src/fights.ts

```diff
@@ -1428,6 +1428,63 @@ const freeFightSources = [
     },
     true
   ),
+  // Use NC forcers to complete shadow quests
+  new FreeFight(
+    () => {
+      if (!have($item`closed-circuit pay phone`)) return false;
+      if (globalOptions.prefs.yachtzeechain) return false; // NCs are better when yachtzeeing, probably
+
+      if (get("rufusQuestType") === "items") {
+        return false; // NCs don't help with item quests
+      }
+
+      // Check NC-forcers
+      if (get("encountersUntilSRChoice") === 0) {
+        // NC is forced natively
+        return true;
+      }
+
+      if (have($item`Clara's bell`) && !globalOptions.clarasBellClaimed) {
+        return true;
+      }
+
+      if (have($item`Jurassic Parka`) && get("_spikolodonSpikeUses") < 5) {
+        return true;
+      }
+
+      return false; // Can't force any NCs for shadow waters
```

Comment:
We should return a `number` here instead of a `boolean`

Reply:
The previous shadow code returns booleans here too, so I think it's okay to keep them?